backend/src/Designer/Services/Implementation/DeploymentService.cs

```diff
@@ -83,29 +83,29 @@ await _applicationInformationService
         }
 
         /// <inheritdoc/>
+        /// TODO: Property DeployedInEnv should not be part of the entity that is stored in db, this property is redundant and unnecessary, as it does not provide any useful information for the database. Moreover, we are returning the DeploymentEntity object directly from the controller, DTO should be used instead.
```

Comment:
Consider creating an issue for this, and add link to issue to this TODO - so that we remember to do this 😊

Reply:
Added link to issue